### DISCUSSION TITLE
有道词典笔S6测试选项

### DISCUSSION THREAD

**Discussion by Cyrene2008** [PenUniverse/PenMods#258] [奇技淫巧]
最近在设置中无意间开启了测试选项，估计是工厂测试用的，但需要密码
打开方式：
设置>关于>存储空间，对着存储占用条  系统 部分连续按十次，返回到上一级，拉到底部即可（会出现在重置选项下面）
经过测试，S6Pro也可以用相同方式开启测试选项

**Comment by SkySight-666** [comment]
S6的测试选项里面其实没有什么有用的东西
X5的测试选项没有密码，参考那个
需要密码的话可以IDA设置的应用包的相应二进制文件，有很明显的明文字符串

**Comment by PenOS-S6Pro** [comment]
密码为password007

**Reply by Cyrene2008** [reply]
实测X7也是相同密码

**Reply by yoyorgznxt** [reply]
实测a7pro也可以

**Reply by wyxdlz54188** [reply]
不是哥们，怎么传到这了

**Reply by xiao-k233** [reply]
已废

**Reply by yoyorgznxt** [reply]
> 已废

已经不行了？

**Reply by xiao-k233** [reply]
> > 已废
> 
> 已经不行了？

yes，密码变了
